conf/defaults.ini

```diff
@@ -429,6 +429,14 @@ default_home_dashboard_path =
 # Upper limit of data sources that Grafana will return. This limit is a temporary configuration and it will be deprecated when pagination will be introduced on the list data sources API.
 datasource_limit = 5000
 
+[datasources.ip_range_security]
+# Set to true to enable IP range access control for Grafana Cloud data sources
+enabled = false
+# Populated by Grafana Cloud. Do not edit.
+allow_list =
+# Populated by Grafana Cloud. Do not edit.
+secret_key =
+
```

Comment:
I don't think you need to add it to defaults.ini if it's not user facing

Reply:
Thanks for the correction. I wasn't sure if we should add all settings to the ini file or not. Will remove it.